product_uom_po_domain/models/product_template.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2021 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    uom_category_id = fields.Many2one(
+        comodel_name="uom.category", related="uom_id.category_id")
+
+    uom_measure_type = fields.Selection(
+        related="uom_id.category_id.measure_type")
```

Comment:
useful ?

Reply:
not for this module, but as a technical field that can be used for other purpose in the same spirit as ``uom_category_id``. I see it a little heavy to have two modules for two fields. But if you think it's not acceptable, I'll create another module for this related field.
Let me know.

Reply:
It's ok for me. Just wanted the why.

PS: this field has been dropped in v14.

Reply:
> PS: this field has been dropped in v14.

Yes. It's a pity, because it is usefull.